src/detection_linux.cpp

```diff
@@ -168,8 +168,8 @@ static ListResultItem_t* GetProperties(struct udev_device* dev, ListResultItem_t
 	}
 	item->vendorId = strtol(udev_device_get_sysattr_value(dev,"idVendor"), NULL, 16);
 	item->productId = strtol(udev_device_get_sysattr_value(dev,"idProduct"), NULL, 16);
-	item->deviceAddress = 0;
-	item->locationId = 0;
+	item->deviceAddress = strtol(udev_device_get_sysattr_value(dev,"devnum"), NULL, 10);
+	item->locationId = strtol(udev_device_get_sysattr_value(dev,"busnum"), NULL, 10);
```

Comment:
Reference to others doing something similar, https://github.com/OpenXT/vusb-daemon/blob/ed721b3caa4b0fc93a54fac6b86cad17228ce842/src/udev.c#L383-L387

It's using `10` because the `devnum` and `busnum` are base 10 vs the others are base 16

> ```
> What:		/sys/bus/usb/devices/usbX/busnum
> Description:
> 		Number of the bus.
> 
> [...]
> 
> What:		/sys/bus/usb/devices/usbX/devnum
> Description:
> 		USB interface device number, in decimal.
> 
> [...]
> 
> What:		/sys/bus/usb/devices/usbX/idProduct
> Description:
> 		Product ID, in hexadecimal.
> 
> What:		/sys/bus/usb/devices/usbX/idVendor
> Description:
> 		Vendor ID, in hexadecimal.
> ```
>
> *-- https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-bus-usb*